tools/cosmovisor/args.go

```diff
@@ -145,6 +158,51 @@ func (cfg *Config) CurrentBin() (string, error) {
 	return binpath, nil
 }
 
+// GetConfigFromFile will read the configuration from the file at the given path.
+// It will return an error if the file does not exist or if the configuration is invalid.
+// If ENV variables are set, they will override the values in the file.
+func GetConfigFromFile(filePath string) (*Config, error) {
+	if filePath == "" {
+		return nil, ErrEmptyConfigENV
```

Comment:
I don't think we should return an error, that will be a behavior for existing users. Maybe we should use a temporary empty file? Quite hacky, but at least it will preserve the current behavior for users that won't use the config file.

Reply:
Make sense.

Updated. Now if the `filePath` is not provided instead of returning an error it will try to load config from `env variables`